### ISSUE TITLE
Treat 'First ... Last' and 'Last, First ...' the same

### ISSUE THREAD

**Issue by salutis** [bdarcus/citar#532]
Some test cases:

- `author = {Knuth, Donald Ervin}` = `Knuth`
- `author = {Knuth, Donald E.}` = `Knuth`
- `author = {Donald Ervin Knuth}` = `Knuth`
- `author = {Donald E. Knuth}` = `Knuth`
- `author = {Donald Knuth}` = `Knuth`
- `author = {D. Knuth}` = `Knuth`
- `author = {Knuth}` = `Knuth`
- `author = {{Knuth, Not}}` = `Knuth, Not`
- `author = {{Not Knuth}}` = `Not Knuth`

**Comment by aikrahguzar** [comment]
> Notes, for whoever gets to looking at this more closely.
> 
> Right now, `citar-display-transform-functions` strips the braces from names before `citar-shorten-names` sees them.
> 
> So that would need refactoring.
This part seems simple, we can just change the order of two functions is the `citar-display-transform-functions`, that way `citar-clean-string` will run after `citar-shorten-names`.

**Comment by bdarcus** [comment]
> This part seems simple, we can just change the order of two functions is the `citar-display-transform-functions`, that way `citar-clean-string` will run after `citar-shorten-names`.

So then that would mean implementing this change, and then updating `citar-shorten-names` to account for braces; right?

**Comment by aikrahguzar** [comment]
> So then that would mean implementing this change, and then updating `citar-shorten-names` to account for braces; right?
Yes. Only the part of letting `citar-shorten-names` take braces into account is simple.

**Comment by bdarcus** [comment]
Update on:

joost just updated parsebib to version 4, which includes this enhancement.

https://github.com/joostkremers/parsebib/commit/83a77ea7e51f78093b1a0f1eb51615ec9295829d

That solves #535, but am not sure yet how to do with this in that context.
